operatorconfig/moduleconfig/common/version-values.yaml

```diff
@@ -10,3 +10,4 @@ powerscale:
   v2.4.0:
       authorization: "v1.4.0"
       replication: "v1.3.0"
+      observability: "v1.3.0"
```

Comment:
different obs modules have different versions, shall we use more fine-graded key name? e.g. obs-topology?

Reply:
@forrestxia this is observability module version 1.3.0, rather than individual component image version

Reply:
observability module version is an abstract concept-only version. If so, shall we use v1.4.0 for the coming release? @YianZong

Reply:
@forrestxia We add v1.3.0 first, then add v1.4.0 for Q4 release.

Reply:
ok, see. thanks for explanation.